### PR TITLE
feat: add ability to disable error stacktrace logging and daily sets that are unavailable in some markets

### PR DESCRIPTION
**Pull request by meghrathod**
By default stacktrace is printed for all users which can be annoying and difficult to debug any issues for normal users. Adding a flag to enable or disable stacktrace through environment variable will be highly appreciated. Instead I've added print statements whenever exceptions occur with proper details about the exception.

Apart from that in India and many other markets, where I come from there is no daily sets and which results in error about the buttons not found so made it an optional flag with default value true.

Let me know your feedback on the commits or merge them if they seem appropriate.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -1096,9 +1127,11 @@ def pc_search_helper(driver, EMAIL, PASSWORD, PC_SEARCHES):
...
     except Exception as e:
         # Print the traceback and sleep for 500 seconds
-        print(traceback.format_exc())
-        print('Attempting to restart PC search in 500 seconds')
-        sleep(500)
+        if(LOGGING):
+            print(traceback.format_exc())
+        print('PC Search failed.')
+        print('Attempting to restart PC search in 100 seconds')
+        sleep(100)
```
Comment (meghrathod) [inline main.py:1134]: 500 seconds for restarting was looking too long for actual restart as well as for diagonosis of problems

Comment (Prem-ium) [reply main.py:1134]: 500 seconds is there for manual intervention from the user. The rest of your pull request looks good, change the sleep time back a& I'll merge.

**Review by Prem-ium — CHANGES_REQUESTED** [review]
Change Sleep Time in case of failure back to original amount.

Otherwise, great work.

**Comment by meghrathod** [comment]
Done. Kindly merge, I still feel a shorter restart time is good for unattended execution when deployed on cloud for example. Maybe we can add a flag for this as well.

**Comment by Prem-ium** [comment]
Instead of using LOGGING, we can rename your new env to DEBUGGING which, if enabled, prints the traceback and pauses for a longer period of time allowing enough time of debugging on execution. If not enabled, we can have the script close the driver earlier as you suggested.

**Review by Prem-ium — APPROVED** [review]
Ready to merge. Good work.

**Pull request merged**